geopyspark/rdd.py

```diff
@@ -1,6 +1,8 @@
-from geopyspark.geotrellis.constants import RESAMPLE_METHODS, NEARESTNEIGHBOR, ZOOM, FLOAT
 import json
 
+from geopyspark.constants import RESAMPLE_METHODS, NEARESTNEIGHBOR, ZOOM, FLOAT, TILE
```

Comment:
Sorry for implication, this should go into `geopyspark/geotrellis` along with `constants`